charts/consul/templates/server-acl-init-job.yaml

```diff
@@ -304,10 +304,6 @@ spec:
             -partition-token-file=/vault/secrets/partition-token \
             {{- end }}
 
-            {{- if .Values.apiGateway.enabled }}
-            -api-gateway-controller=true \
```

Comment:
Did we not have any spots in the actual job binary that were using stanza or was it all in the templating?